### DISCUSSION TITLE
GitHub no longer works in visual studio 2022

### DISCUSSION THREAD

**Discussion by Ian-Beard** [community/community#37618] [Other Feature Feedback, Questions, & Ideas]
I've been told I have to run this form a command prompt:
git config --global --add safe.directory c:/Users/***/source/repos/***
I use visual studio 2022, I do not have a git command prompt.
When I tried to install the power shell version it told me it was unsafe to install so I aborted the installation.
I am now looking for an alternate source code management system as the git plugin for 2022 refuses to allow me to use my paid for repository.
My confidence is git is now ZERO.

**Comment by ossanna16** [comment]
Hi there @Ian-Beard and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
